starboard/shared/uwp/media_is_video_supported.cc

```diff
@@ -242,7 +242,7 @@ class XboxVideoCapabilities {
     bool is_supported = hw_decoder_capabilities_.IsSupported(
         codec, transfer_id, width, height, fps);
 
-    if (ExtendedResourcesManager::GetInstance()->IsGpuDecoderReady() &&
+    if (ExtendedResourcesManager::GetInstance()->WaitGpuDecoderReady() &&
```

Comment:
There are times when the this takes very long, and typically even at the times when it's working at "normal" speed it takes long enough to make resuming the app while it is playing a video too sluggish (especially when switching between a game and Cobalt).

In some cases it actually never becomes ready for reasons outside of our control, and in other cases the GPU decoder itself does not finish initialization, such as when texture allocation fails due to memory fragmentation.

AFAICS, this change will (at least) cause the app to block before doing any playback, causing delay in the playback and potentially causing the app to hang indefinitely.

The choice was made on purpose to not wait and fallback to non-GPU accelerated playback until the GPU decoders are available, if ever.

Reply:
Ok, I'm completely agree. The player has much more high priority than tests. I rollbacked all changes and added few extra calls of SbMediaIsVideoSupported  in tests to reduce error probability. The other solution suggests to change SbMediaIsVideoSupported interface by adding "bool wait_results = false" to it. I'm not sure this will be the best now